syntax = "proto3";

package example;

message EventNotification {
	int32 unique_client_id = 1;
	float timestamp = 2;
	int32 message_number = 3;
	string client_name = 4;
    float metric_0 = 5;
	float metric_1 = 6;
	float metric_2 = 7;
	float metric_3 = 8;
	float metric_4 = 9;
	float metric_5 = 10;
	float metric_6 = 11;
	float metric_7 = 12;
	float metric_8 = 13;
	float metric_9 = 14;
	float metric_10 = 15;
	float metric_11 = 16;
}